src/dropsgear.ts

```diff
@@ -454,11 +453,8 @@ function juneCleaver(equipMode: BonusEquipMode): Map<Item, number> {
   }
   if (!juneCleaverEV) {
     juneCleaverEV =
-      JuneCleaver.choices.reduce(
-        (total, choice) =>
-          total +
-          valueJuneCleaverOption(juneCleaverChoiceValues[choice][bestJuneCleaverOption(choice)]),
-        0
+      sum([...JuneCleaver.choices], (choice) =>
+        valueJuneCleaverOption(juneCleaverChoiceValues[choice][bestJuneCleaverOption(choice)])
```

Comment:
i don't have any suggestions for improvements but this is... quite the statement lol. takes a little bit of noodling to figure out what's going on with the [] references and the functions. i don't necessarily think it needs a change but wanted to at least note it.

Reply:
Yeah I would love to get better names for these the second someone else comes up with them